gui_qt/workspace_setup_dialog.py

```
@@ -248,10 +390,143 @@ def _refresh_plan(self) -> None:
+            "选择 Ren'Py SDK 目录（需包含 renpy.py）",
+            str(start),
+        )
+        if not picked:
+            return
+        path = Path(canonical_abs_path(picked))
+        self._browse_path_edit.setText(_display_path(path))
+        if is_renpy_sdk_dir(str(path)):
+            self._sdk_browse.setChecked(True)
+            self._sdk_status.setText("已选择有效 SDK 目录。")
+        else:
+            self._sdk_status.setText("所选目录不是有效 SDK（缺少 renpy.py）。")
```

Ruff (0.15.21)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)